WeatherTests/UnitTests/CityValidatorTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherBL.Validators;
+
+namespace WeatherTests.UnitTests
+{
+    [TestFixture]
+    public class CityValidatorTests
+    {
+        private CityValidator cityValidator;
```

Comment:
private readonly 
and lets name local variables with _, example _cityValidator

Reply:
fixed